client/my-sites/marketing/do-it-for-me/difm-landing.tsx

```diff
@@ -240,7 +261,7 @@ export default function DIFMLanding( {
 	const difmTieredPriceDetails = getDIFMTieredPriceDetails( product );
 	const extraPageCost = difmTieredPriceDetails?.perExtraPagePrice;
 
-	const currencyCode = useSelector( getCurrentUserCurrencyCode );
+	const currencyCode = useSelector( ( state ) => getProductCurrencyCode( state, WPCOM_DIFM_LITE ) );
```

Comment:
Switched to this selector because the currency code is already available in this store. When using the `getCurrentUserCurrencyCode` selector, the data needs to be fetched again by `<QueryProductsList />`.

Reply:
Wondering if you could elaborate more on this.

Are we trying to move away from `QueryProductsList` usage? I noticed it's still implemented in https://github.com/Automattic/wp-calypso/pull/83300/files#diff-653af82b386dfd83fa33be98cb3cbfd370c8bc0765c974c85422a06f62543d31R56, so it looks like the data is being fetched anyway?

Reply:
This was a good hint! Since `QueryProductsList` is already called inside `DIFMLanding`, the one in the `new-or-existing-site` step was not required. I have removed it via bc8c2fc01d243b8f1975daa2b2c2f6c53db018a7.